src/check-imports/packages-not-allowed.ts

```diff
@@ -12,7 +12,6 @@ const notAllowed: NameAndRange[] = [
   ['@aws-sdk/credential-provider-ini', '>3.193.0'],
   ['@aws-sdk/credential-provider-node', '>3.193.0'],
   ['@aws-sdk/smithy-client', '>3.193.0'],
-  ['@aws-sdk/types', '>3.193.0'],
```

Comment:
should we remove the other aws-sdk dependencies too?

Reply:
I don't think we can because aws-nock doesn't support some of the changes that sdk > 3.193 introduces. 
I think we need to fix those before we push this higher.